packages/SwingSet/test/test-comms.js

```diff
@@ -96,7 +96,7 @@ test('receive', t => {
   // look at machine B, which is receiving remote messages aimed at a local
   // vat's object 'bob'
   const { syscall, sends } = mockSyscall();
-  const d = buildCommsDispatch(syscall, 'fakestate', 'fakehelpers');
+  const d = buildCommsDispatch(syscall, 'fakestate', 'fakehelpers', {});
```

Comment:
using a default arg in `buildCommsDispatch` should remove the need for these extra `{}`

Reply:
done